frontend/src/metabase/visualizations/visualizations/CartesianChart/CartesianChart.tsx

```diff
@@ -130,7 +131,10 @@ function _CartesianChart(props: VisualizationProps) {
     setChartSize({ width, height });
   }, []);
 
-  const canSelectTitle = !!onChangeCardAndRun;
+  // We can't navigate a user to a particual card from a visualizer viz,
+  // so title selection is disabled in this case
+  const canSelectTitle = !!onChangeCardAndRun && !isVisualizerViz;
```

Comment:
Wouldn't it be cleaner to _not_ pass `onChangeCardAndRun` as a way to express that the title isn't clickable, maybe?

Reply:
Sorry, not sure I understand, I don't think we're passing it? [Here's the line in code](https://github.com/metabase/metabase/blob/4c4024608ba995df43d34439a4c0de56bb512252/frontend/src/metabase/visualizations/components/ChartCaption.tsx#L51)